core/src/test/resources/log4j4ibis.xml

```diff
@@ -2,7 +2,7 @@
 <log4j2:Configuration xmlns:log4j2="log4j-config.xsd" name="${ctx:instance.name}-test">
 	<Appenders>
 		<Console name="stdout">
-			<PatternLayout pattern="%d{HH:mm:ss,SSS} %-5p %c{1}.%M():%L %m%n"/>
+			<PatternLayout pattern="%d{HH:mm:ss,SSS} %-5p %c{1}.%M():%L %x%m%n"/>
```

Comment:
Let wel op dat wij de ThreadContext in de NDC pattern stijl weergeven. Zie de [IbisNdcPatternConverter](https://github.com/ibissource/iaf/blob/master/core/src/main/java/nl/nn/adapterframework/logging/IbisNdcPatternConverter.java) voor meer informatie. Het log patroon hiervoor is `%TC`.